UI/window-basic-settings.cpp

```diff
@@ -3409,6 +3417,25 @@ void OBSBasicSettings::SaveAudioSettings()
 		main->UpdateVolumeControlsPeakMeterType();
 	}
 
+	if (WidgetChanged(ui->meterUpdateRate)) {
+		uint32_t meterUpdateRate;
+		switch (ui->meterUpdateRate->currentIndex()) {
+		case 0:
```

Comment:
For readability, may be worth moving this case down alongside default, or removing it altogether.

Reply:
I followed the example of the Meter Decay Rate setting, which has a case for 0 as well as default.

Reply:
Why would we combine case 0 (30Hz selected in UI) with default (60Hz selected in UI)?  Did you mean combining case 1 and default?  I think what Matt's suggesting this:
```
		switch (ui->meterUpdateRate->currentIndex()) {
		case 0:
			meterUpdateRate = VOLUME_METER_UPDATE_30HZ;
			break;
		case 1:
		default:
			meterUpdateRate = VOLUME_METER_UPDATE_60HZ;
			break;
		}
```

Because the results of selecting 60Hz or having some unknown index are both that we want to set OBS to update the meter at 60 Hz.

Reply:
What I mean is that the example I followed that already existed in code had separate cases for the same outcome (i.e. case 0 and default), and did not combine them: https://github.com/obsproject/obs-studio/blob/master/UI/window-basic-settings.cpp#L3385-L3396

It would make sense to combine them, though, sure.

Reply:
I'd guess that they weren't combined in the code for meter decay rate because it would ruin the 0-1-2-default ordering of the cases (you'd end up with 1-2-0-default or simply because the code author preferred explicit and separate case blocks.  We don't really have a specific requirement for making it one way or the other, and I don't have a strong preference.

Reply:
I went ahead and combined the cases anyway.